### PR TITLE
test: Add ABRT packages into fedora-26

### PR DESCRIPTION
**Pull request by marusak**
Add ABRT packages to enable testing ABRT support in Fedora 26 Alpha release.
Blocks #6189.

### CODE REVIEW

File: test/images/scripts/fedora-26.setup [hunk 1]
```diff
@@ -70,6 +70,15 @@ gdb \
 targetcli \
 "
 
+# For correct work of ABRT in Fedora 26 Alpha release a following packages
```
Comment (stefwalter) [inline test/images/scripts/fedora-26.setup]: Lets mark this with ```HACK:``` and a link to the bugzilla URL that solves the issue long term.

Comment (marusak) [inline test/images/scripts/fedora-26.setup]: Updated

**Comment by stefwalter** [comment]
bot: Image refresh for fedora-26

**Comment by cockpituous** [comment]
Image creation for fedora-26 in process on cockpit-11.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-26-4b08a60cb47728d3b9d3d4eab97c7d38ba8313c4/

**Comment by cockpituous** [comment]
Image creation failed.

**Comment by mvollmer** [comment]
Seems that `mock --installdeps` on Fedora 26 is now broken, see #6438

**Comment by marusak** [comment]
it seems that https://github.com/cockpit-project/cockpit/pull/6438 was resolved. Can we continue on this and then on https://github.com/cockpit-project/cockpit/pull/6189?

**Comment by mvollmer** [comment]
bot: Image refresh for fedora-26

**Comment by cockpituous** [comment]
Image creation for fedora-26 in process on cockpit-tests-9mch4.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-26-4b08a60cb47728d3b9d3d4eab97c7d38ba8313c4/

**Comment by mvollmer** [comment]
bot: Image refresh for fedora-26

**Comment by cockpituous** [comment]
Image creation for fedora-26 in process on cockpit-tests-6b6pv.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-26-4b08a60cb47728d3b9d3d4eab97c7d38ba8313c4/

**Comment by cockpituous** [comment]
Image creation failed.

**Comment by marusak** [comment]
I guess we need to rebase it, don't we? I did so, now should the image creation succeeded.

**Comment by mvollmer** [comment]
> I guess we need to rebase it, don't we? I did so, now should the image creation succeeded.

Yes.  I thought that the bot will rebase, but it only does that for testing, not image creation, apparently.

**Comment by mvollmer** [comment]
bot: Image refresh for fedora-26

**Comment by cockpituous** [comment]
Image creation for fedora-26 in process on cockpit-tests-cv4f6.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-26-93353a9122a2cdfe06797d2f13e1ab7035b81982/

**Comment by cockpituous** [comment]
Image creation failed.

**Comment by mvollmer** [comment]
Hrr, uploading the image failed:

```
Permission denied (publickey).
rsync: connection unexpectedly closed (0 bytes received so far) [sender]
rsync error: unexplained error (code 255) at io.c(226) [sender=3.1.2]
```

**Comment by mvollmer** [comment]
bot: Image refresh for fedora-26

**Comment by cockpituous** [comment]
Image creation for fedora-26 in process on verifymachine4.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-26-93353a9122a2cdfe06797d2f13e1ab7035b81982/

**Comment by cockpituous** [comment]
Image creation for fedora-26 in process on cockpit-7.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-26-93353a9122a2cdfe06797d2f13e1ab7035b81982/

**Comment by mvollmer** [comment]
> Hrr, uploading the image failed:

SSH config has been fixed.

**Comment by cockpituous** [comment]
Image creation aborted

**Comment by cockpituous** [comment]
Image creation aborted

**Comment by mvollmer** [comment]
bot: Image refresh for fedora-26

**Comment by cockpituous** [comment]
Image creation for fedora-26 in process on cockpit-tests-x9nmw.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-26-93353a9122a2cdfe06797d2f13e1ab7035b81982/

**Comment by cockpituous** [comment]
Image creation for fedora-26 in process on cockpit-tests-1fp3r.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-26-93353a9122a2cdfe06797d2f13e1ab7035b81982/

**Comment by cockpituous** [comment]
Image creation for fedora-26 in process on cockpit-tests-ljg4z.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-26-93353a9122a2cdfe06797d2f13e1ab7035b81982/

**Comment by cockpituous** [comment]
Image creation for fedora-26 in process on cockpit-7.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-fedora-26-93353a9122a2cdfe06797d2f13e1ab7035b81982/

**Comment by mvollmer** [comment]
Woah, the bots are really eager today, and the mechanism for aborting in case of conflict doesn't seem to work very well... sorry for all the hassle.

**Comment by cockpituous** [comment]
Image creation aborted

**Comment by cockpituous** [comment]
Image creation aborted

**Comment by cockpituous** [comment]
Image creation aborted

**Comment by cockpituous** [comment]
Image creation aborted

**Comment by mvollmer** [comment]
bot: Image refresh for fedora-26

**Comment by marusak** [comment]
I am not sure after all that retries if the creation was successful or not. How does it look with the image?

**Comment by mvollmer** [comment]
No image has been created by the bots.  I make one by hand, and debug the bots a bit.

**Comment by stefwalter** [comment]
bot: Image refresh for fedora-26

**Comment by stefwalter** [comment]
> Woah, the bots are really eager today, and the mechanism for aborting in case of conflict doesn't seem to work very well... sorry for all the hassle.

Yeah, that is something I noticed too. I have a WIP pull request to generalize that so it can properly cover image creation too:

https://github.com/cockpit-project/cockpituous/pull/116

**Comment by stefwalter** [comment]
@mvollmer Due to fedorapeople.org deleting images, we had to replace the image stuff in short order. In order for you to upload an image, I need the following from you (in an email):

```
$ htpasswd -nb user $(cat ~/.config/github-token)
```

**Comment by stefwalter** [comment]
@marusak In order to move this forward I've added a commit with a built image. That image is currently uploading. I had to force push in order to make this work due to conflicts with fedora-26 changes on git master.

**Comment by marusak** [comment]
@stefwalter thank you!

**Pull request merged**